index.ts

```diff
@@ -0,0 +1,60 @@
+import express from "express";
+import bodyParser from "body-parser";
+import {
+  deleteUser,
+  getUsers,
+  postUser,
+  putUser,
+} from "./controller/controller";
+import { port } from "./infrastructureVariables";
+const app = express();
+
+app.use(bodyParser.urlencoded());
+
+app.get("/", async (req, res) => {
+  try {
+    const users = await getUsers();
+    res.type("json").send(users);
+  } catch (err) {
+    res.status(500).send(err);
+  }
```

Comment:
Move logic with statuses to controller.

Reply:
Logic with statuses has been moved to the controller